apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/RetryConformanceTest.cs

```diff
@@ -421,14 +346,5 @@ private static StringContent GetBodyContent(string methodName, InstructionList i
         return new StringContent(body.ToString(), Encoding.UTF8, "application/json");
     }
 
-    /// <summary>
-    /// Function created to be used for getting environmental variables
-    /// </summary>
-    private static string GetEnvironmentVariableOrDefault(string name, string defaultValue)
-    {
-        string value = Environment.GetEnvironmentVariable(name);
-        return string.IsNullOrEmpty(value) ? defaultValue : value;
-    }
-
     private static void SleepAfterBucketCreateDelete() => Thread.Sleep(2000);
```

Comment:
nit: Should this now be just `SleepAfterBucketCreate()` as we have removed the bucket deletion code?